modules/mageck/count/main.nf

```diff
@@ -0,0 +1,48 @@
+
+process MAGECK_COUNT {
+    tag "$meta.id"
+    label 'process_medium'
+
+
+    conda (params.enable_conda ? "bioconda::mageck=0.5.9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mageck:0.5.9--py37h6bb024c_0':
+        'quay.io/biocontainers/mageck:0.5.9--py37h6bb024c_0' }"
+
+    input:
+    tuple val(meta), path(inptfile)
+    path(library)
+    val(name)
```

Comment:
Can this be passed via meta and modules.config? Usually trying to keep the input section to files if possible :)